python/tests/test_regression.py

```diff
@@ -0,0 +1,41 @@
+from vw_executor.vw import Vw
```

Comment:
this file seems not related to regression but more of a core infrasturcture.
Better to rename it to "core.py/test.py/..."